lua/net.lua

```diff
@@ -3,6 +3,21 @@ local ffi = require("ffi")
 
 local M = {}
 
+function ipv4_to_num(addr)
```

Comment:
I ended up passing the IPv4 string to C to then call `inet_pton()`. Is that what you had in mind?